frontend/src/pages/mainPage.js

```diff
@@ -39,28 +64,22 @@ function MainPage({news}) {
                 <div className="streamsBar">
 
                     <div className="activeStreams">
-
                         <h2 id="liveStreams">Активні</h2>
 
-                        <div className="streamBox">
-
-                            <i className="fa fa-user-o" aria-hidden="true"></i>
-
-                            <div className="streamInfo">
-
-                                <h4 className="online">етер</h4>
-
-                                <h1 className="score">0:0</h1>
-
-                                <h4 className="matchLeague">ліга</h4>
-
-                            </div>
-
-                            <i className="fa fa-user-o" aria-hidden="true"></i>
+                        {games.slice(0, 5).map((item, index) => (
```

Comment:
Hardcoded values are now welcome unless crucial and required  (I saw a couple places with same values used in the same way).
Consider moving it to some global variable or something like that

Reply:
That was a test slice by @Romlen2, he's trying to get correct path to the required data